libs/dyn/convert/from_typed.go

```diff
@@ -39,6 +39,10 @@ func FromTyped(src any, ref dyn.Value) (dyn.Value, error) {
 // Private implementation of FromTyped that allows for additional options not exposed
 // in the public API.
 func fromTyped(src any, ref dyn.Value, options ...fromTypedOptions) (dyn.Value, error) {
+	if src == nil {
+		return dyn.NilValue, nil
+	}
```

Comment:
When does this happen?

Reply:
This is when `vint := v.Interface()` later at L108 returns nil. I believe it might happen when default or value for variable is not defined but can't remember exactly

Reply:
@pietern maybe I'm missing something but what's concern with return NilValue if nil is source? it seems to be straightforward and it allows supports conversion of nil values, in maps for instance, correctly

Reply:
I think this is the same/similar to #1518, as the variable value type is now `any`.

Reply:
Indeed, works now without it, removed